### PR TITLE
uniongang.tv

### PR DESCRIPTION
**Pull request by galeksandrp**
Issue author: @galeksandrp

### CODE REVIEW

File: src/chrome/content/rules/uniongang.tv.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+mail.uniongang.tv ⁴
+uniongang.tv mixed content
+www.uniongang.tv mixed content
+
+
+⁴ self signed
+-->
+<ruleset name="uniongang.tv (partial)">
+	<target host="forum.uniongang.tv" />
```
Comment (J0WI) [inline src/chrome/content/rules/uniongang.tv.xml]: This has also mixed content. Could you add the `platform` and the other MCB targets?

**Comment by 6u621jpq6ybdw** [comment]
@galeksandrp you only need to add platform="mixedcontent"

**Comment by galeksandrp** [comment]
Confirm, working on.

**Comment by galeksandrp** [comment]
Added.

**Pull request merged**